### DISCUSSION TITLE
Download preferred quality

### DISCUSSION THREAD

**Discussion by Wuerger** [Flexget/Flexget#3932] [Q&A] [answered]
I prefer to download shows in 1080p, but sometimes that quality for an episode is not available. How can I configure the series plugin to download an episode in a lower quality if it can't find it in 1080p? The bound parameters seem to select _all_ qualities listed and don't seem to have conditions... Thanks!

**Comment by BrutuZ** [comment, marked as answer]
https://flexget.com/Plugins/timeframe
